src/basic/cgroup-util.c

```diff
@@ -199,10 +199,8 @@ int cg_rmdir(const char *controller, const char *path) {
                 return -errno;
 
         r = cg_hybrid_unified();
-        if (r < 0)
+        if (r <= 0)
```

Comment:
huh, i wonder if it's worth checking for more of these with coccinelle.

Reply:
there's only one way to find out: #10803